moped-editor/src/views/projects/projectView/NoteInputQuill.js

```diff
@@ -0,0 +1,147 @@
+import React, { useRef, useEffect } from "react";
+import {
+  Box,
+  Button,
+  Container,
+  Grid,
+  FormControl,
+  Radio,
+  RadioGroup,
+  FormLabel,
+  FormControlLabel,
+} from "@mui/material";
+import ReactQuill from "react-quill";
+import "react-quill/dist/quill.snow.css";
+import makeStyles from "@mui/styles/makeStyles";
+import ProjectSaveButton from "../newProjectView/ProjectSaveButton";
+
+const quillModules = {
+  toolbar: [
+    ["bold", "italic", "underline", "strike"],
+    [{ list: "ordered" }, { list: "bullet" }],
+    ["link"],
+    ["clean"],
+  ],
+};
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    width: "100%",
+    backgroundColor: theme.palette.background.paper,
+  },
+  cancelButton: {
+    margin: theme.spacing(1),
+    position: "relative",
+    color: theme.palette.secondary.dark,
+  },
+  quillText: {
+    color: theme.palette.text.primary,
+  },
+  showButtonItem: {
+    margin: theme.spacing(2),
+  },
+}));
+
+/**
+ * Defines the NoteTypeRadioButtons that allow a user to select the note type of a
+ * note they are creating or editing.
+ * @param {integer} defaultValue - The note type id that determines which button option
+ * should be selected/highlighted by default
+ * @param {function} onChange - Callback function that runs when the note type is changed via button click
+ * @return {JSX.Element}
+ * @constructor
+ */
+const NoteTypeRadioButtons = ({ defaultValue, onChange }) => (
+  <RadioGroup
+    row
+    value={defaultValue}
+    onChange={onChange}
+    sx={{ color: "black" }}
```

Comment:
couldnt for the life of me figure out why/where the editing mode was inheriting grey text for the radio button labels, so this is the fix i came up withhh. i feel like its a bandaid solution and not great though?

Reply:
I don't think its a bandaid necessarily, the styles are being inherited from two different places, so its good to specify a consistent color.  I dont know why the top one is using Mui-Paper-root-Mui-Card-root, but the edit one is using a typography style, even tho they are both in cards?

Reply:
agreed :)